packages/ckeditor5-image/tests/imagestyle/imagestylecommand.js

```diff
@@ -402,6 +402,18 @@ describe( 'ImageStyleCommand', () => {
 					expect( getData( model ) )
 						.to.equal( initialData );
 				} );
+
+				it( 'should not set width and height when command `setImageSizes` parameter is false', async () => {
+					const initialData = '<paragraph>[<imageInline src="/assets/sample.png"></imageInline>]</paragraph>';
```

Comment:
I'd add a test for block images and also tests for `ImageTypeCommand`.

Reply:
Added. Also some missing tests for adding width and height automatically in image type command and image style command.